.github/workflows/test.yml

```diff
@@ -10,7 +10,7 @@ jobs:
     strategy:
       matrix:
         python-version: ['3.7', '3.8', '3.9']
-        django: [3.1.3, 3.0.2, 2.2]
+        django: [3.1.4, 3.0.11, 2.2]
```

Comment:
I think we should add patch version to 2.2 too.

Reply:
let's do that :)

Reply:
Done.